website/source/layouts/guides.erb

```diff
@@ -50,6 +50,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("guides-tls") %>>
+        <a href="/guides/tls.html">Configuring TLS</a>
```

Comment:
not a **strong** opinion, just a mild one: 😄 
_anything would be fine as long as the keywords on a web search will give this page to a newbie searching for configuration options._
keywords: TLS, secure, security guide, SSL, certificate, etc. etc.

Keep up the great work! 👍 👍